app/models/elastic_custom_client.rb

```diff
@@ -0,0 +1,19 @@
+require 'multi_json'
+require 'faraday'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.